src/styles/global.css

```
@@ -1262,6 +1262,127 @@ button {
+  width: 100%;
+  max-width: 100%;
+  place-items: start center;
+}
+
+.article-content .mermaid-diagram-image {
+  display: block;
+  width: auto;
+  max-width: 100%;
+  max-height: calc(
+    var(--mermaid-diagram-max-height)
+    - var(--mermaid-diagram-padding)
```

Stylelint (17.14.0)

[error] Unexpected newline before "-" (scss/operator-no-newline-before)

(scss/operator-no-newline-before)